Makefile

```diff
@@ -266,3 +271,13 @@ tools/bin/misspell:tools/check/go.mod
 tools/bin/ineffassign:tools/check/go.mod
 	cd tools/check; \
 	$(GO) build -o ../bin/ineffassign github.com/gordonklaus/ineffassign
+
+# Usage:
+#
+# 	$ make vectorized-bench VB_FILE=Time VB_FUNC=builtinCurrentDateSig
+vectorized-bench:
+	cd ./expression && \
+		go test -v -benchmem \
+			-bench=BenchmarkVectorizedBuiltin$(VB_FILE)Func \
+			-run=BenchmarkVectorizedBuiltin$(VB_FILE)Func \
+			-args "$(VB_FUNC)"
```

Comment:
If VB_FUNC is unset, it should run all test, but runs no tests now. Could you take a look?
Maybe you can modify these two lines.

https://github.com/pingcap/tidb/blob/cf06460c11d41741beab927428b885fc16d5a930/expression/bench_test.go#L709

https://github.com/pingcap/tidb/blob/cf06460c11d41741beab927428b885fc16d5a930/expression/bench_test.go#L866

Reply:
Sure, will fix it.

Reply:
Done.

<details>
  <summary>Example</summary>

```sh
➜ make vectorized-bench VB_FILE=Time
cd ./expression && \
	go test -v -benchmem \
		-bench=BenchmarkVectorizedBuiltinTimeFunc \
		-run=BenchmarkVectorizedBuiltinTimeFunc \
		-args ""
goos: linux
goarch: amd64
pkg: github.com/pingcap/tidb/expression
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDatetimeAndDurationSig-VecBuiltinFunc-12         	   19498	     62119 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDatetimeAndDurationSig-NonVecBuiltinFunc-12      	   12296	     96693 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDatetimeAndStringSig-VecBuiltinFunc-12           	     891	   1378775 ns/op	   97629 B/op	    5940 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDatetimeAndStringSig-NonVecBuiltinFunc-12        	     836	   1462300 ns/op	   97698 B/op	    5940 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDurationAndDurationSig-VecBuiltinFunc-12         	  226000	      5154 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDurationAndDurationSig-NonVecBuiltinFunc-12      	   41254	     29174 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDurationAndStringSig-VecBuiltinFunc-12           	     976	   1269900 ns/op	   95933 B/op	    5805 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDurationAndStringSig-NonVecBuiltinFunc-12        	     918	   1289646 ns/op	   95592 B/op	    5805 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddStringAndDurationSig-VecBuiltinFunc-12           	     661	   1731741 ns/op	  200035 B/op	    9760 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddStringAndDurationSig-NonVecBuiltinFunc-12        	     656	   1830534 ns/op	  199343 B/op	    9760 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddStringAndStringSig-VecBuiltinFunc-12             	     448	   2610017 ns/op	  286713 B/op	   14947 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddStringAndStringSig-NonVecBuiltinFunc-12          	     440	   2622399 ns/op	  286689 B/op	   14947 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDateAndDurationSig-VecBuiltinFunc-12             	    3534	    339708 ns/op	   95432 B/op	    3756 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDateAndDurationSig-NonVecBuiltinFunc-12          	    3194	    373863 ns/op	   95435 B/op	    3756 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDateAndStringSig-VecBuiltinFunc-12               	     666	   1812350 ns/op	  214530 B/op	   10613 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddDateAndStringSig-NonVecBuiltinFunc-12            	     661	   1869706 ns/op	  213955 B/op	   10613 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddTimeDateTimeNullSig-VecBuiltinFunc-12            	17585562	        69.9 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddTimeDateTimeNullSig-NonVecBuiltinFunc-12         	   95511	     12807 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddTimeStringNullSig-VecBuiltinFunc-12              	  518574	      2200 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddTimeStringNullSig-NonVecBuiltinFunc-12           	  217180	      5552 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddTimeDurationNullSig-VecBuiltinFunc-12            	18499635	        69.0 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFuncGenerated/builtinAddTimeDurationNullSig-NonVecBuiltinFunc-12         	  123720	      8566 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimeFormatSig-VecBuiltinFunc-12                              	   10000	    107083 ns/op	   34860 B/op	    1300 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimeFormatSig-NonVecBuiltinFunc-12                           	   10000	    125717 ns/op	   34860 B/op	    1300 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinAddStringAndStringSig-VecBuiltinFunc-12                      	  428689	      2980 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinAddStringAndStringSig-NonVecBuiltinFunc-12                   	   51592	     23616 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinMonthNameSig-VecBuiltinFunc-12                               	   35272	     35196 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinMonthNameSig-NonVecBuiltinFunc-12                            	   20811	     57017 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinYearSig-VecBuiltinFunc-12                                    	   66490	     18158 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinYearSig-NonVecBuiltinFunc-12                                 	   27596	     43317 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinCurrentDateSig-VecBuiltinFunc-12                             	  219404	      5762 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinCurrentDateSig-NonVecBuiltinFunc-12                          	    6786	    177702 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp1ArgSig-VecBuiltinFunc-12                           	    3111	    405244 ns/op	  147457 B/op	    7168 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp1ArgSig-NonVecBuiltinFunc-12                        	    2769	    449047 ns/op	  147457 B/op	    7168 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp1ArgSig-VecBuiltinFunc#01-12                        	    3186	    360308 ns/op	  147457 B/op	    7168 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp1ArgSig-NonVecBuiltinFunc#01-12                     	    3226	    410093 ns/op	  147457 B/op	    7168 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp1ArgSig-VecBuiltinFunc#02-12                        	     351	   3075029 ns/op	  576666 B/op	   16282 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp1ArgSig-NonVecBuiltinFunc#02-12                     	     308	   4106757 ns/op	  576654 B/op	   16282 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp1ArgSig-VecBuiltinFunc#03-12                        	     380	   2965873 ns/op	  459072 B/op	   10361 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp1ArgSig-NonVecBuiltinFunc#03-12                     	     278	   3808126 ns/op	  459072 B/op	   10361 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp2ArgsSig-VecBuiltinFunc-12                          	     477	   2574039 ns/op	  295647 B/op	   15146 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp2ArgsSig-NonVecBuiltinFunc-12                       	     444	   2516384 ns/op	  295383 B/op	   15146 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp2ArgsSig-VecBuiltinFunc#01-12                       	    2023	    563790 ns/op	  114923 B/op	    5572 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp2ArgsSig-NonVecBuiltinFunc#01-12                    	    1770	    703358 ns/op	  147945 B/op	    7168 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp2ArgsSig-VecBuiltinFunc#02-12                       	     397	   3244483 ns/op	  475200 B/op	   10725 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinTimestamp2ArgsSig-NonVecBuiltinFunc#02-12                    	     283	   3987038 ns/op	  475200 B/op	   10725 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinMonthSig-VecBuiltinFunc-12                                   	   65761	     17995 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinMonthSig-NonVecBuiltinFunc-12                                	   27534	     43236 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinUTCDateSig-VecBuiltinFunc-12                                 	 1000000	      1119 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinUTCDateSig-NonVecBuiltinFunc-12                              	    5385	    217551 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinDateSig-VecBuiltinFunc-12                                    	   53418	     22578 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinDateSig-NonVecBuiltinFunc-12                                 	   19036	     61182 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinDayOfMonthSig-VecBuiltinFunc-12                              	   72123	     16665 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinDayOfMonthSig-NonVecBuiltinFunc-12                           	   28725	     41953 ns/op	       0 B/op	       0 allocs/op
PASS
ok  	github.com/pingcap/tidb/expression	76.352s

```
</details>